kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/Tab8021xUi.java

```diff
@@ -53,20 +46,17 @@ public class Tab8021xUi extends Composite implements NetworkTab {
 
     private static Tab8021xUiUiBinder uiBinder = GWT.create(Tab8021xUiUiBinder.class);
 
-    private final GwtSecurityTokenServiceAsync gwtXSRFService = GWT.create(GwtSecurityTokenService.class);
-    private final GwtNetworkServiceAsync gwtNetworkService = GWT.create(GwtNetworkService.class);
-
     private static final Messages MSGS = GWT.create(Messages.class);
 
-    private static final Logger logger = Logger.getLogger(Tab8021xUi.class.getSimpleName());
+    private static final String PASSWORD_PLACEHOLDER = "Password-Placeholder-123";
```

Comment:
We already have a placeholder in the Type you are using for saving the password: https://github.com/eclipse/kura/blob/f82c49030cd29f19250315e617d47b43e5955f00/kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/NewPasswordInput.java#L30